Modules/Common/include/Common/TH1Ratio.inl

```diff
@@ -19,18 +19,38 @@
 namespace o2::quality_control_modules::common
 {
 
+template<class T>
+TH1Ratio<T>::TH1Ratio()
+  : T(),
+    o2::mergers::MergeInterface(),
+    mUniformScaling(true)
+{
+  {
+    TDirectory::TContext ctx(nullptr);
```

Comment:
This is "inspired" from [TEfficiency](https://root.cern.ch/doc/master/TEfficiency_8cxx_source.html).
I guess this is done because the numerator and denominator histograms are saved together with the container object, and therefore should not be also written independently.
I can write a line of comment with the link.

Reply:
Thanks!